### ISSUE TITLE
After installing with curl, how to completely remove PDM?

### ISSUE THREAD

**Issue by robs-nice99** [pdm-project/pdm#2572]
Hello,

I installed PDM using curl. Now I would like to remove it. How can  I do that?

**Comment by frostming** [comment]
```
curl -sSL https://pdm-project.org/install-pdm.py | python3 - --remove
```

**Comment by robs-nice99** [comment]
I still see a lot of files in `~/.local/bin` that I think were installed by pdm, right?

**Comment by frostming** [comment]
> I still see a lot of files in `~/.local/bin` that I think were installed by pdm, right?

No, the folder is holding executable scripts, and pdm only ships one `pdm` to it.
